ruby/lib/gherkin3/parser.rb

```diff
@@ -63,9 +63,12 @@ def initialize(ast_builder=AstBuilder.new)
       @ast_builder = ast_builder
     end
 
-    def parse(token_scanner, token_matcher=TokenMatcher.new)
+    def parse(input, token_matcher=TokenMatcher.new)
```

Comment:
This is inconsistent with the other implementations. parse(scanner) should work too.